--- stone_soup-0.6.0/source/rltiles/Makefile~	2010-03-27 00:43:25.000000000 +0100
+++ stone_soup-0.6.0/source/rltiles/Makefile	2010-04-09 14:51:04.752345526 +0200
@@ -82,6 +82,6 @@
 	$(QUIET_CXX)$(CXX) $(CFLAGS) -Wp,-MD,$*.d,-MT,$@ -c $< -o $@
 
 $(TILEGEN): $(OBJECTS)
-	$(QUIET_LINK)$(CXX) $(OBJECTS) -o $@ $(LDFLAGS)
+	$(QUIET_LINK)$(CXX) $(OBJECTS) -o $@ $(LDFLAGS) `pkg-config libpng --libs` `pkg-config sdl --libs` -lSDL_image
 
 .PHONY: all clean distclean
