--- stone_soup-0.5.1-src/source/makefile.unix~	2009-08-12 15:50:27.000000000 +0200
+++ stone_soup-0.5.1-src/source/makefile.unix	2009-08-12 15:50:06.000000000 +0200
@@ -53,8 +53,8 @@
 LEX  := flex
 YACC := bison -y
 
-LUASRC := util/lua/src
-LUALIB = lua
+LUASRC := /usr/include/lua51 
+LUALIB = lua51
 LUALIBA = l$(LUALIB).a
 
 SQLSRC   := util/sqlite
@@ -66,7 +66,7 @@
 EXTRA_INCLUDES += -I$(SQLSRC)
 EXTRA_DEPENDS  += $(FSQLLIBA)
 
-LIB = -l$(LIBCURS) -L$(LUASRC) -l$(LUALIB) $(LIBDBM)
+LIB = -l$(LIBCURS) -l$(LUALIB) $(LIBDBM)
 
 INCLUDES := $(INCLUDES) -Iutil -I. -I$(LUASRC) $(EXTRA_INCLUDES)
 
@@ -135,7 +135,7 @@
 DOYACC :=
 endif
 
-GAME_DEPENDS := $(LUASRC)$(LUALIBA) $(EXTRA_DEPENDS) $(OBJECTS)
+GAME_DEPENDS := $(EXTRA_DEPENDS) $(OBJECTS)
 SRC_PKG_BASE := stone_soup
 SRC_VERSION  := $(shell egrep 'VER_NUM  *".*"' version.h | \
 			egrep -o '[0-9]\.[0-9](\.[0-9])?')
@@ -278,9 +278,9 @@
 #############################################################################
 # Build Lua
 
-$(LUASRC)$(LUALIBA):
-	echo Building Lua...
-	cd $(LUASRC) && $(MAKE) crawl_unix
+#$(LUASRC)$(LUALIBA):
+#	echo Building Lua...
+#	cd $(LUASRC) && $(MAKE) crawl_unix
 
 #############################################################################
 # Build SQLite
--- stone_soup-0.5.1-src/source/makefile_tiles.unix~	2009-07-29 01:31:27.000000000 +0200
+++ stone_soup-0.5.1-src/source/makefile_tiles.unix	2009-08-13 16:42:51.000000000 +0200
@@ -68,8 +68,8 @@
 LEX  := flex
 YACC := bison -y
 
-LUASRC := util/lua/src
-LUALIB = lua
+LUASRC := /usr/include/lua51
+LUALIB = lua51
 LUALIBA = l$(LUALIB).a
 
 SQLSRC   := util/sqlite
@@ -101,7 +101,7 @@
 
 EXTRA_DEPENDS += $(DESTTILEFILES)
 
-LIB = -L$(LUASRC) -l$(LUALIB) $(LIBDBM) -lpng $(SDL_LDFLAGS) -lGL -lGLU -lSDL_image -lfreetype
+LIB = -l$(LUALIB) $(LIBDBM) -lpng $(SDL_LDFLAGS) -lGL -lGLU -lSDL_image -lfreetype
 
 INCLUDES := $(INCLUDES) -Iutil -I. -I$(LUASRC) $(EXTRA_INCLUDES) $(PNG_INCLUDE) $(FREETYPE_INCLUDE)
 
@@ -142,7 +142,7 @@
 DOYACC :=
 endif
 
-GAME_DEPENDS := $(LUASRC)$(LUALIBA) $(EXTRA_DEPENDS) $(OBJECTS)
+GAME_DEPENDS := $(EXTRA_DEPENDS) $(OBJECTS)
 SRC_PKG_BASE := stone_soup
 SRC_VERSION  := $(shell egrep 'VER_NUM  *".*"' version.h | \
 			egrep -o '[0-9]\.[0-9](\.[0-9])?')
@@ -312,9 +312,9 @@
 #############################################################################
 # Build Lua
 
-$(LUASRC)$(LUALIBA):
-	echo Building Lua...
-	cd $(LUASRC) && $(MAKE) crawl_unix
+#$(LUASRC)$(LUALIBA):
+#	echo Building Lua...
+#	cd $(LUASRC) && $(MAKE) crawl_unix
 
 #############################################################################
 # Build SQLite
