--- stone_soup-0.5.1-src/source/makefile.unix~	2009-08-12 15:24:25.000000000 +0200
+++ stone_soup-0.5.1-src/source/makefile.unix	2009-08-12 15:33:02.000000000 +0200
@@ -57,14 +57,14 @@
 LUALIB = lua51
 LUALIBA = l$(LUALIB).a
 
-SQLSRC   := util/sqlite
+SQLSRC   := /usr/include
 SQLLIB   := sqlite3
 SQLLIBA  := lib$(SQLLIB).a
 FSQLLIBA := $(SQLSRC)/$(SQLLIBA)
 
-LIBDBM         := -L$(SQLSRC) -l$(SQLLIB)
-EXTRA_INCLUDES += -I$(SQLSRC)
-EXTRA_DEPENDS  += $(FSQLLIBA)
+LIBDBM         := -l$(SQLLIB)
+EXTRA_INCLUDES +=
+EXTRA_DEPENDS  +=
 
 LIB = -l$(LIBCURS) -l$(LUALIB) $(LIBDBM)
 
@@ -275,9 +275,9 @@
 #############################################################################
 # Build SQLite
 
-$(FSQLLIBA):
-	echo Building SQLite
-	cd $(SQLSRC) && $(MAKE)
+#$(FSQLLIBA):
+#	echo Building SQLite
+#	cd $(SQLSRC) && $(MAKE)
 
 #############################################################################
 # Packaging a source tarball for release
