--- stone_soup-0.6.0/source/makefile~	2010-03-27 00:43:25.000000000 +0100
+++ stone_soup-0.6.0/source/makefile	2010-04-09 12:21:00.676580838 +0200
@@ -44,7 +44,7 @@
 #    ask for a package with convenience libraries instead -- we'll try to provide
 #    them somewhere in the near future.
 
-GAME = crawl
+GAME = stone_soup
 
 #
 # Compiler Flags
@@ -78,11 +78,11 @@
 
 else
 
-CFOPTIMIZE := -O2
+CFOPTIMIZE := 
 
 endif # USE_ICC
 
-CFOTHERS := -fno-strict-aliasing -pipe $(EXTERNAL_FLAGS)
+CFOTHERS := -pipe $(EXTERNAL_FLAGS)
 CFOTHERS_L := -fsigned-char
 CFWARN := -Wall
 
@@ -659,7 +659,7 @@
 
 CFLAGS   := $(CFOPTIMIZE) $(CFOTHERS) $(CFWARN)
 CFLAGS_L := $(CFOPTIMIZE_L) $(DEFINES_L) $(CFWARN_L) $(INCLUDES_L) $(CFOTHERS_L)
-ALL_CFLAGS := $(CFLAGS) $(CFLAGS_L)
+ALL_CFLAGS := $(OPTFLAGS) $(CFLAGS) $(CFLAGS_L)
 YACC_CFLAGS  := $(ALL_CFLAGS) -w -DYYENABLE_NLS=0 -DYYLTYPE_IS_TRIVIAL=0
 
 UTIL = util/
