--- stone_soup-0.6.0/source/makefile~	2010-04-09 13:32:53.939477805 +0200
+++ stone_soup-0.6.0/source/makefile	2010-04-09 13:33:39.472831138 +0200
@@ -387,9 +385,9 @@
 endif
 
 ifndef BUILD_LUA
-  ifneq (,$(wildcard /usr/include/lua5.1))
-    INCLUDES_L += -I/usr/include/lua5.1
-    LIBS += -llua5.1
+  ifneq (,$(wildcard /usr/include/lua51))
+    INCLUDES_L += -I/usr/include/lua51
+    LIBS += -llua51
   else
     ifneq (,$(wildcard /usr/include/lua.h))
       LIBS += -llua
