--- stone_soup-0.7.0/source/makefile~	2010-07-24 14:16:48.204232436 +0200
+++ stone_soup-0.7.0/source/makefile	2010-07-24 14:19:27.494635214 +0200
@@ -45,7 +45,7 @@
 #    ask for a package with convenience libraries instead -- we'll try to provide
 #    them somewhere in the near future.
 
-GAME = crawl
+GAME = stone_soup
 
 ASSERTS = yes
 
@@ -86,7 +86,7 @@
 
 else
 
-CFOPTIMIZE := -O2
+CFOPTIMIZE :=
 
 endif # USE_ICC
 
@@ -727,7 +727,7 @@
 
 CFLAGS   := $(CFOPTIMIZE) $(CFOTHERS) $(CFWARN)
 CFLAGS_L := $(CFOPTIMIZE_L) $(DEFINES_L) $(CFWARN_L) $(INCLUDES_L) $(CFOTHERS_L)
-ALL_CFLAGS := $(CFLAGS) $(CFLAGS_L)
+ALL_CFLAGS := $(OPTFLAGS) $(CFLAGS) $(CFLAGS_L)
 YACC_CFLAGS  := $(ALL_CFLAGS) -w -DYYENABLE_NLS=0 -DYYLTYPE_IS_TRIVIAL=0
 
 UTIL = util/
