--- stone_soup-0.4.1-src/source/makefile.unix	2008-07-18 01:35:53.000000000 +0200
+++ stone_soup-0.4.1-src/source/makefile.unix.mod	2008-07-19 12:15:36.968770818 +0200
@@ -50,8 +50,8 @@
 LEX  := flex
 YACC := bison -y
 
-LUASRC := util/lua/src
-LUALIB = lua
+LUASRC := /usr/include/lua51 
+LUALIB = lua51
 LUALIBA = l$(LUALIB).a
 
 DBH_FILE  := /usr/include/db.h
@@ -85,7 +85,7 @@
 EXTRA_DEPENDS  += $(FSQLLIBA)
 endif
 
-LIB = -l$(LIBCURS) -L$(LUASRC) -l$(LUALIB) $(LIBDBM)
+LIB = -l$(LIBCURS) -L/usr/lib -l$(LUALIB) $(LIBDBM)
 
 INCLUDES := $(INCLUDES) -Iutil -I. -I$(LUASRC) $(EXTRA_INCLUDES)
 
@@ -156,7 +156,7 @@
 DOYACC :=
 endif
 
-GAME_DEPENDS := $(LUASRC)$(LUALIBA) $(EXTRA_DEPENDS) $(OBJECTS)
+GAME_DEPENDS := $(EXTRA_DEPENDS) $(OBJECTS)
 SRC_PKG_BASE := stone_soup
 SRC_VERSION  := $(shell egrep 'VER_NUM  *".*"' version.h | \
 			egrep -o '[0-9]\.[0-9](\.[0-9])?')
